src/cli/grain.js

```diff
@@ -21,7 +21,10 @@ function die(std, message) {
 }
 
 const grainCommand: Command = async (args, std) => {
-  if (args.length !== 0) {
+  let simulation = false;
+  if (args.length === 1 && args[0] === "--sim") {
```

Comment:
Recommend making it more descriptive with an optional abbreviation, e.g. let the flag be `--simulate` or `-s` for short.